features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScopeTest.kt

```diff
@@ -1506,6 +1565,99 @@ internal class RumSessionScopeTest {
+        whenever(mockView.viewId) doReturn fakeViewId
+        whenever(mockChildScope.activeView) doReturn mockView
+
+        fakeDatadogContext = fakeDatadogContext.copy(
+            featuresContext = fakeDatadogContext.featuresContext.toMutableMap().apply {
+                put(Feature.SESSION_REPLAY_FEATURE_NAME, mapOf(fakeViewId to mapOf("has_replay" to fakeHasReplay)))
+            }
+        )
+
+        val info = RumTTIDInfo(
+            scenario = scenario,
+            durationNs = forge.aLong(min = 0, max = 10000)
+        )
+
+        val event = RumRawEvent.AppStartTTIDEvent(
+            info = info
+        )
+
+        // When
+
+        val result = testedScope.handleEvent(
+            event = event,
+            datadogContext = fakeDatadogContext,
+            writeScope = mockEventWriteScope,
+            writer = mockWriter
+        )
+
+        // Then
+        val context = checkNotNull(result).getRumContext()
+
+        argumentCaptor<VitalEvent> {
+            verify(mockWriter).write(eq(mockEventBatchWriter), capture(), eq(EventType.DEFAULT))
+            VitalEventAssert.assertThat(lastValue).apply {
+                hasDate(scenario.initialTime.timestamp + fakeTimeInfo.serverTimeOffsetMs)
+                hasApplicationId(context.applicationId)
+                containsExactlyContextAttributes(fakeParentAttributes)
+                hasStartReason(context.sessionStartReason)
+                hasSampleRate(100f)
+                hasNoSyntheticsTest()
+                hasSessionId(context.sessionId)
+                hasSessionType(fakeRumSessionType?.toVital() ?: VitalEvent.VitalEventSessionType.USER)
+                hasSessionReplay(fakeHasReplay)
+                hasNullView()
+                hasSource(fakeVitalSource)
+                hasAccountInfo(fakeDatadogContext.accountInfo)
+                hasUserInfo(fakeDatadogContext.userInfo)
+                hasDeviceInfo(
+                    fakeDatadogContext.deviceInfo.deviceName,
+                    fakeDatadogContext.deviceInfo.deviceModel,
+                    fakeDatadogContext.deviceInfo.deviceBrand,
+                    fakeDatadogContext.deviceInfo.deviceType.toVitalSchemaType(),
+                    fakeDatadogContext.deviceInfo.architecture
+                )
+                hasOsInfo(
+                    fakeDatadogContext.deviceInfo.osName,
+                    fakeDatadogContext.deviceInfo.osVersion,
+                    fakeDatadogContext.deviceInfo.osMajorVersion
+                )
+                hasConnectivityInfo(fakeDatadogContext.networkInfo)
+                hasVersion(fakeDatadogContext.version)
+                hasServiceName(fakeDatadogContext.service)
+                hasDDTags(buildDDTagsString(fakeDatadogContext))
+            }
+
+            val vital = lastValue.vital
+            check(vital is VitalEvent.Vital.AppLaunchProperties)
+
+            VitalAppLaunchPropertiesAssert.assertThat(vital).apply {
```

Comment:
minor: should work as well if you move `VitalAppLaunchPropertiesAssert.assertThat` to import and then it will be just `assertThat` here.